src/types/session.ts

```diff
@@ -1,12 +1,32 @@
-// required attributes
 import * as t from "io-ts";
 import { UTCISODateFromString } from "italia-ts-commons/lib/dates";
+import { FiscalCode } from "italia-ts-commons/lib/strings";
 import { SessionToken } from "./token";
 
-export const Session = t.interface({
+const Session = t.interface({
   createdAt: UTCISODateFromString,
+  deletedAt: t.union([UTCISODateFromString, t.null], "DeletionDate"),
   expirationTime: UTCISODateFromString,
+  fiscalCode: FiscalCode,
   token: SessionToken
 });
 
-export type Session = t.TypeOf<typeof Session>;
+type Session = t.TypeOf<typeof Session>;
+
+export const OpenSession = t.intersection([
+  Session,
+  t.interface({
+    deletedAt: t.null
+  })
+]);
+
+export type OpenSession = t.TypeOf<typeof OpenSession>;
+
+export const CloseSession = t.intersection([
```

Comment:
add a comment describing what is this (perhaps it should be named `ClosedSession` or better `ExpiredSession`?)